Source/WebCore/platform/ScrollbarTheme.h

```diff
@@ -51,7 +51,7 @@ class ScrollbarTheme {
     virtual bool paint(Scrollbar&, GraphicsContext&, const IntRect& /*damageRect*/) { return false; }
     virtual ScrollbarPart hitTest(Scrollbar&, const IntPoint&) { return NoPart; }
     
-    virtual int scrollbarThickness(ScrollbarControlSize = ScrollbarControlSize::Regular, ScrollbarExpansionState = ScrollbarExpansionState::Expanded) { return 0; }
+    virtual int scrollbarThickness(ScrollbarControlSize = ScrollbarControlSize::Regular, bool = false, ScrollbarExpansionState = ScrollbarExpansionState::Expanded) { return 0; }
```

Comment:
The bool param needs a name (commented out, since the implementation is empty). Or it might be cleaner to just pass the ScrollbarWidth enum.

Reply:
Given that auto and thin are accounted for in the controls size, none could maybe be added in that?

 Or alternatively we could just pass scrollbar width and not control size but then we'd need to do the mapping in multiple places?

Happy to just pass down the ScrollbarWidth value though.

Reply:
That makes sense I will replace it with the ScrollbarWidth enum. A follow up change could probably remove the ScrollbarControlSize enum entirely as ScrollbarWidth should be able to replace it entirely

Reply:
You can do it separately.